go.mod

```diff
@@ -2,7 +2,11 @@ module github.com/rancher/rancher-operator
 
 go 1.15
 
-replace k8s.io/client-go => k8s.io/client-go v0.20.0
+replace (
+	github.com/rancher/rancher/pkg/apis => github.com/pennyscissors/rancher/pkg/apis v0.0.0-20210205233934-6d0193d42bf0
+	github.com/rancher/rancher/pkg/client => github.com/pennyscissors/rancher/pkg/client v0.0.0-20210205233934-6d0193d42bf0
```

Comment:
These are just placeholders to make sure I'm vendoring the right thing. They will be replaced once [this PR](https://github.com/rancher/rancher/pull/31138) is merged.